[Update] Self-hosted Configuration pages (#2116)
timescale/docs: self-hosted/configuration/configuration.md

`SHOW config_file;` from your PostgreSQL client (for example, `psql`).
</Highlight>

In addition, other Timescale specific settings can be modified through the

> `postgresql.conf` file as discussed in our section about [Timescale settings][ts-settings]

Reviewer: ```suggestion
`postgresql.conf` file as covered in the [Timescale settings][ts-settings] section.
```

### Disk-write settings

In order to increase write throughput, there are
[multiple settings][async-commit] to adjust the behavior that PostgreSQL uses

> to write data to disk. We find the performance to be good with the default (safest)

Reviewer: ```suggestion
to write data to disk. In tests, performance is good with the default, or safest,
```

review the official PostgreSQL documentation on
[lock management][lock-management].

## Timescale configuration and tuning

Just as you can tune settings in PostgreSQL, Timescale provides a number of
configuration settings that may be useful to your specific installation and
performance needs. These can also be set within the `postgresql.conf` file or as
command-line parameters when starting PostgreSQL.

### Policies

#### `timescaledb.max_background_workers (int)`

> Max background worker processes allocated to Timescale.  Set to at

Reviewer: ```suggestion
Max background worker processes allocated to Timescale. Set to at
```

## Changing configuration with Docker

When running Timescale in a [Docker container][docker], there are
two approaches to modifying your PostgreSQL configuration. In the
following example, we modify the size of the database instance's
write-ahead-log (WAL) from 1&nbsp;GB to 2&nbsp;GB in a Docker container named
`timescaledb`.

#### Modifying postgres.conf inside Docker

> 1.  Get into a shell in Docker in order to change the configuration on a running

Reviewer: ```suggestion
1.  Open a shell in Docker to change the configuration on a running
```
